mobileapp/README.md

```diff
@@ -1,40 +1,54 @@
-Requires docker build (from project folder / parent repository)
-```
-docker-compose build public_web 
-```
+# Mobileapp
+
+##
+
+A few prerequisite:
+
+- iOS: install cocoapods
+- Run `npm install -g cordova@9.0.0 code-push-cli@2.1.9`
+- Download `GoogleService-Info.plist` from Firebase (any alternative for contributors?)
```

Comment:
@FelixDQ is this correct?

Reply:
I don't think it's required for running it locally. I don't remember downloading this, but I might just be forgetting it. It looks like I've wiped my repo since last time I got it running :/

Reply:
ping me if you need it

Reply:
> ping me if you need it

I just downloaded it from Firebase. But I wanted to document that contributors might not be able to build the app. Is that correct?

> I don't think it's required for running it locally.

Or it might be related to my xcode version. 🤔